docs/BR.md

```diff
@@ -1699,59 +1698,61 @@ By issuing the Certificate, the CA represents that it followed the procedure set
-
-i. Certificate Field: subject:organizationalUnitName (OID: 2.5.4.11)
-Optional.
-The CA SHALL implement a process that prevents an OU attribute from including a name, DBA, tradename, trademark, address, location, or other text that refers to a specific natural person or Legal Entity unless the CA has verified this information in accordance with Section 3.2 and the Certificate also contains subject:organizationName, subject:givenName, subject:surname, subject:localityName, and subject:countryName attributes, also verified in accordance with Section 3.2.2.1.
+a. __Certificate Field:__ `subject:commonName` (OID 2.5.4.3)  
+   __Required/Optional:__ Deprecated (Discouraged, but not prohibited)  
+   __Contents:__ If present, this field MUST contain a single IP address or Fully-Qualified Domain Name that is one of the values contained in the Certificate's subjectAltName extension (see Section 7.1.4.2.1).
+
+b. __Certificate Field:__ `subject:organizationName` (OID 2.5.4.10)  
+   __Required/Optional:__ Optional.  
+   __Contents:__ If present, the `subject:organizationName` field MUST contain either the Subject's name or DBA as verified under Section 3.2.2.2. The CA may include information in this field that differs slightly from the verified name, such as common variations or abbreviations, provided that the CA documents the difference and any abbreviations used are locally accepted abbreviations; e.g., if the official record shows "Company Name Incorporated", the CA MAY use "Company Name Inc." or "Company Name". Because Subject name attributes for individuals (e.g. givenName (2.5.4.42) and surname (2.5.4.4)) are not broadly supported by application software, the CA MAY use the `subject:organizationName` field to convey a natural person Subject's name or DBA.
+
+c. __Certificate Field:__ `subject:givenName` (2.5.4.42) and `subject:surname` (2.5.4.4)  
+   __Required/Optional:__ Optional.  
+   __Contents:__ If present, the `subject:givenName` field and `subject:surname` field MUST contain an natural person Subject’s name as verified under Section 3.2.3. A Certificate containing a `subject:givenName` field or `subject:surname` field MUST contain the (2.23.140.1.2.3) Certificate Policy OID.
+
+d. __Certificate Field:__ Number and street: `subject:streetAddress` (OID: 2.5.4.9)  
+   __Required/Optional:__ Optional if the `subject:organizationName` field, `subject:givenName` field, or `subject:surname` field are present. Prohibited if the `subject:organizationName` field, `subject:givenName`, and `subject:surname` field are absent.  
```

Comment:
This seems like a bit of a regression compared with the BRs (https://cabforum.org/wp-content/uploads/CA-Browser-Forum-BR-1.6.8.pdf ), which emphasized **Required** / **Optional** via bold and also had them on separate lines.

Reply:
I'm definitely wrestling with this one. You're absolutely right, I'm just trying to figure out how to fix it. This is a lettered list, so it's done manually rather than as a real Markdown list—as a result, trying to get indentations underneath items is fiddly, because Markdown doesn't like doing indent/outdent stuff (one place where something like reStructuredText or AsciiDoc might be more powerful). Here's one option that's probably the simplest to implement:
![SC26-opt1](https://user-images.githubusercontent.com/1791326/76453228-dce6f300-63a8-11ea-8d8b-c100a6a1357a.png)
I'd want more indents on the Optional/Required thing, but I can't figure out a way to get them without blowing up the Contents line after them. Or we could re-org and put Contents before Required/Optional, so those are hanging off the bottom, _comme ça:_
![opt2](https://user-images.githubusercontent.com/1791326/76453572-7b735400-63a9-11ea-8119-5c4307fa23a7.png)

Reply:
Oh! I see what you mean (and recording it here for others who may ask):

If this was a numbered list, Markdown would naturally support the aligned indent by aligning the text on the second line to the first text on the opening line. That is:

```
1. Foo
    Bar
    Baz
``` 
which results in:

1. Foo
    Bar
    Baz

However, because we're using a lettered list with a non-monospaced font, the whitespace will be variable (and not aligned). That's because Markdown sees this as "just" text.

Could you capture in a comment what your proposed syntax would be, both code-quoted and without? I'm curious how the GitHub UI is going to interpret things.

Reply:
Sure! Here's option 1:
_snip!_
d. __Certificate Field:__ Number and street: `subject:streetAddress` (OID: 2.5.4.9)  
   __Required/Optional:__  
   __Optional__ if the `subject:organizationName` field, `subject:givenName` field, or `subject:surname` field are present.  
   __Prohibited__ if the `subject:organizationName` field, `subject:givenName`, and `subject:surname` field are absent.  
   __Contents:__ If present, the `subject:streetAddress` field MUST contain the Subject's street address information as verified under Section 3.2.2.1.
_/snip!_

```
d. __Certificate Field:__ Number and street: `subject:streetAddress` (OID: 2.5.4.9)  
   __Required/Optional:__  
   __Optional__ if the `subject:organizationName` field, `subject:givenName` field, or `subject:surname` field are present.  
   __Prohibited__ if the `subject:organizationName` field, `subject:givenName`, and `subject:surname` field are absent.  
   __Contents:__ If present, the `subject:streetAddress` field MUST contain the Subject's street address information as verified under Section 3.2.2.1.
```

Here's option 2:

_snip!_
d. __Certificate Field:__ Number and street: `subject:streetAddress` (OID: 2.5.4.9)  
   __Contents:__ If present, the `subject:streetAddress` field MUST contain the Subject's street address information as verified under Section 3.2.2.1.  
   __Required/Optional:__
   - __Optional__ if the `subject:organizationName` field, `subject:givenName` field, or `subject:surname` field are present.
   - __Prohibited__ if the `subject:organizationName` field, `subject:givenName`, and `subject:surname` field are absent.  
_/snip!_
```
d. __Certificate Field:__ Number and street: `subject:streetAddress` (OID: 2.5.4.9)  
   __Contents:__ If present, the `subject:streetAddress` field MUST contain the Subject's street address information as verified under Section 3.2.2.1.  
   __Required/Optional:__
   - __Optional__ if the `subject:organizationName` field, `subject:givenName` field, or `subject:surname` field are present.
   - __Prohibited__ if the `subject:organizationName` field, `subject:givenName`, and `subject:surname` field are absent.  
```

Reply:
Option 1 matches [our existing PDF](https://cabforum.org/wp-content/uploads/CA-Browser-Forum-BR-1.6.7.pdf#page=52)

Note that in the PDF/Word doc version, only one field has a "Required/Optional" and that's subject:commonName ... I'm not even sure what the 'required' is supposed to convey here.

Looking at the other fields in 7.1.4.2.2, they're aligned with your Option 1, with the note that they don't use "Required/Optional" but instead specify, each on a new line, Required, Optional, or Prohibited (e.g. 7.1.4.2.2(f) )

Reply:
Great! Easy enough. a1a20ca fixes this whole section to Option 1, with an additional correction in 8b70d19, and a fix in 671531c to fix a missing backtick marker on item e. :)